README.md

```diff
@@ -338,9 +338,38 @@ original SSN key does not exist in the map the Gonymizer will automatically scra
 Every time gonymizer checks a value in the SSN column it will look up this value and replace it with the previously 
 anonymized SSN. This allows us to map keys between tables.
 
-*Note:* Multiple tables can link back to the user table by simply adding the schema, table, and column names to the 
+**Note 1:** Multiple tables can link back to the user table by simply adding the schema, table, and column names to the 
 parent fields in the map file for the specified column.
 
+You must also add the parent table itself as a parent when creating a relationship mapping. From the example
+above the same would be true:
+
+```
+{
+    "TableSchema": "public",
+    "TableName": "user",
+    "ColumnName": "ssn",
+    "DataType": "integer",
+    "ParentSchema": "public",
+    "ParentTable": "user",
+    "ParentColumn": "ssn",
+    "OrdinalPosition": 6,
+    "IsNullable": false,
+    "Processors": [
+        {
+            "Name": "AlphaNumericScrambler",
+            "Max": 0,
+            "Min": 0,
+            "Variance": 0,
+            "Comment": ""
+        }
+    ]
```

Comment:
```suggestion
    ],
```
My JSON may be off. Do you need a `,` here?

Reply:
good catch